CMakeSettings.json

```diff
@@ -218,11 +203,6 @@
           "name": "BUILD_PERFORMANCE_TESTS",
           "value": "True",
           "type": "BOOL"
-        },
-        {
-          "name": "CMAKE_BUILD_TYPE",
-          "value": "MinSizeRel",
-          "type": "STRING"
```

Comment:
I was seeing similar warnings locally about `CMAKE_BUILD_TYPE` being defined twice. Do you know where this is already defined?

Reply:
@vhvb1989 who added them.  I removed cause they are noise.

Reply:
lol, after removing it, the Perf profiles for VS won't use use the most optimized build. 
As long as you don't use this profiles, how is it noise?